_minutes/README.md

```diff
@@ -10,25 +10,23 @@ After the end of each meeting, meeting notes are published here.
 
 ## Upcoming meetings
 
-- 2023-11-23 at 8 AM PST = https://everytimezone.com/?t=655e9600,3c0
 - 2023-12-07 at 8 AM PST = https://everytimezone.com/?t=65710b00,3c0
+- 2024-01-04 at 8 AM PST = https://everytimezone.com/?t=6595f500,3c0
```

Comment:
@dotproto Note that this is 2024. We should not forget to create new events for the events on the WECG calendar, because the current "last one" is scheduled for the 21st of December. I omitted that because historically we have been skipping the "end of December" meetings.

Reply:
Thanks for the reminder!